travis_sphinx/build.py

```diff
@@ -33,10 +33,21 @@ def build(ctx, source, nowarn):
     """
     _logger.info('building documentation')
     outdir = ctx.obj['outdir']
-    args = ['-b html']
+
+    try:
+        # Location from Sphinx 1.7 on
+        sphinx_build = sphinx.cmd.build.build_main
+        # args have to be specified this way for 1.7+, otherwise one gets
+        # "Builder name  html not registered or available through entry point"
+        args = ['-b', 'html']
+    except AttributeError:
+        # Old location
```

Comment:
Would you prefer an explicit version check?

Reply:
I think this approach seems fine with me